aws_jobs/cyhy-data-extract.py

```diff
@@ -247,15 +293,19 @@ def main():
     if args["--date"]:
         # Note this date is in UTC timezone
         date_of_data = datetime.strptime(args["--date"], "%Y-%m-%d")
-        start_of_data_collection = timezone("UTC").localize(
-            date_of_data
-        ) + relativedelta(days=-1, hour=0, minute=0, second=0, microsecond=0)
-        end_of_data_collection = start_of_data_collection + relativedelta(days=1)
+        end_of_data_collection = flatten_datetime(
+            timezone("UTC").localize(date_of_data)
+        )
+        start_of_data_collection = end_of_data_collection + relativedelta(days=-1)
```

Comment:
This same line is repeated three lines later, so it can be pulled out of the `if` statement and put after it.

Reply:
Resolved in 7f94544